app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,44 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name')
```

Comment:
This field will be empty for an anonymous user's post. In that case we should set it to "A Fandom user" (localized and hopefully existing already in the translation files).

Reply:
I believe we should be able to use `this.getTranslatedMessage('username-anonymous')`.